packages/kit/src/states/jotai/contexts/swap/actions.ts

```
@@ -1416,14 +1422,22 @@
+            fromNetworkDefault?.fromToken.isNative
+          ) {
+            set(swapSelectToTokenAtom(), fromNetworkDefault?.fromToken);
           }
           if (toToken && toToken.networkId !== fromToken.networkId) {
             set(swapSelectToTokenAtom(), undefined);
           }
-        } else if (fromNetworkDefault.fromToken) {
-          set(swapSelectFromTokenAtom(), fromNetworkDefault.fromToken);
+        } else if (
+          fromNetworkDefault?.fromToken &&
+          fromNetworkDefault?.fromToken?.isNative &&
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)